prql-compiler/prqlc/tests/test.rs

```diff
@@ -75,3 +76,1145 @@ fn test_compile() {
     ----- stderr -----
     "###);
 }
+
+#[test]
+fn test_shell_completion() {
+    assert_cmd_snapshot!(Command::new(get_cargo_bin("prqlc")).arg("shell-completion").arg("bash"), @r###"
```

Comment:
Possibly given the length of this, we don't put inline? If we remove the `@""`, insta makes a snapshot file.

Reply:
Thanks, I didn't know that!